src/pages/index.tsx

```diff
@@ -1,20 +1,41 @@
 import Hero from "@Component/Hero";
+import Slide from "@Component/Slide";
 import DATA from "data/ong";
 import img_index from 'assets/img-index.png';
```

Comment:
```suggestion
import img_index from 'assets/img-index.png';
```
Consider the implementation of a new field in DATA to include this image to improve reusability